app/models/organization_timeline_entry.rb

```diff
@@ -37,23 +38,39 @@ def duration
     return DateTime.now.to_i - started_at.to_i
   end
 
+  def already_in_queue?
+    ['structural', 'electrical'].include?(entry_type) &&
+      !organization.organization_timeline_entries.current.send(entry_type).empty?
+  end
+
   #notifcations 
-  after_create :notifyStart
+  after_create :notifyStart, :notify_booth_committee
   after_update :notifyEnd
 
   def notifyStart
-    for chair in organization.booth_chairs
-      if chair.phone_number.length == 10
-        send_sms(chair.phone_number, "Downtime for your organization, " +organization.name+", has started.")
+    if entry_type == 'downtime'
+      for chair in organization.booth_chairs
+        if chair.phone_number.length == 10
+          send_sms(chair.phone_number, "Downtime for your organization, " +organization.name+", has started.")
+        end
       end
     end
   end
 
   def notifyEnd
-    for chair in organization.booth_chairs
-      if chair.phone_number.length == 10
-        send_sms(chair.phone_number, "Downtime for your organization, " +organization.name+", has ended. You have "+Time.at(organization.remaining_downtime).utc.strftime("%H hours %M minutes")+" left.")
+    if entry_type == 'downtime'
+      for chair in organization.booth_chairs
+        if chair.phone_number.length == 10
+          send_sms(chair.phone_number, "Downtime for your organization, " +organization.name+", has ended. You have "+Time.at(organization.remaining_downtime).utc.strftime("%H hours %M minutes")+" left.")
+        end
       end
     end
   end
-end
\ No newline at end of file
+
+  def notify_booth_committee
+    # post in the booth committee groupme when someone joins a queue
+    if ['structural', 'electrical'].include?(entry_type)
+      send_groupme("8a6cab13d71d09562023b48658","#{entry_type.titlecase} Queue: #{organization.short_name} needs #{description}")
```

Comment:
Please notify the electrical groupMe of electrical queue additions here as well.